save-backend/src/main/kotlin/org/cqfn/save/backend/service/OrganizationService.kt

```diff
@@ -10,6 +14,27 @@ import org.springframework.stereotype.Service
  */
 @Service
 class OrganizationService(private val organizationRepository: OrganizationRepository) {
+    /**
+     * Store [organization] in the database
+     *
+     * @param organization an [Organization] to store
+     * @return organization's id, should never return null
+     */
+    @Suppress("UnsafeCallOnNullableType")
+    fun saveOrganization(organization: Organization): Pair<Long, OrganizationSaveStatus> {
+        val exampleMatcher = ExampleMatcher.matchingAll()
+            .withMatcher("name", ExampleMatcher.GenericPropertyMatchers.exact())
+            .withMatcher("ownerId", ExampleMatcher.GenericPropertyMatchers.exact())
```

Comment:
I recognize copy-paste from `ProjectService`, but doesn't `matchingAll` already mean all fields match `exact`? I don't remember, why we use it like this there, but I think it's redundant.

Reply:
Oh yes, there was a copy-paste from `ProjectService`. Now it is something like that:
```
val exampleMatcher = ExampleMatcher.matching()
            .withMatcher("name", ExampleMatcher.GenericPropertyMatchers.exact())
```
But I cannot see what is the difference between `matching` and `matchingAll` cause according to [`docs.spring.io`](https://docs.spring.io/spring-data/commons/docs/current/api/org/springframework/data/domain/ExampleMatcher.html) they do exactly the same.

Reply:
Actually this is what source code looks like:
```
static ExampleMatcher matching() {
	return matchingAll();
}
```

Reply:
I mean, can we simply drop two lines with specific matchers and only leave `matchingAll`? I believe, result will be the same

Reply:
The idea is to match only name because either we get one organization per user (`matchingAny`) or many organizations with same name (`matching`/`matchingAll`)

Reply:
Then we can simply use `findByName`, because example matcher is better suited for different case - when you would like to match against multiple fields

Reply:
Oh ok, will fix it soon

Reply:
Done